File: Java/A1/P3.java
public class P3 {
    public static void main(String[] args) {
        int a = 1, b = 2;
        System.out.print("Before swaping vale of a is " + a);
        System.out.println(" and vale of b is " + b);
        a = a + b;
        b = a - b;
        a = a - b;
        System.out.print("After swaping vale of a is " + a);
        System.out.println(" and vale of b is " + b);
    }
}

File: HTML/A3/P3.php
<!-- Write a PHP code for swapping two numbers using call by value -->
<!DOCTYPE html>
<html>
    <head>
        <title>P3</title>
    </head>
    <body>
        <?php
            function swap($a,$b){
                $temp = $a;
                $a = $b;
                $b = $temp;
                echo"After swapping: a = ".$a." and b = ".$b."<br>";
            }
            $a = 10;
            $b = 20;
            echo"Before swapping: a = ".$a." and b = ".$b."<br>";
            swap($a,$b);
        ?>
    </body>
</html>